src/pages/fr/install/auto.md

```diff
@@ -2,150 +2,100 @@
+  <Fragment slot="yarn">
+  ```shell
+  # create a new project with yarn
+  yarn create astro
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+Vous pouvez exécuter `create-astro` n'importe où sur votre machine, il n'y a donc pas besoin de créer un nouveau répertoire vide  pour votre projet avant de commencer. Si vous n'avez encore pas de répertoire vide pour votre nouveau projet, l'assistant aidera à en créer un automatiquement.
+
+Si tout se passe bien, vous devriez voir un message "Ready for liftoff!" suivi de quelques "Étapes suivantes". Exécutez `cd` vers votre nouveau répertoire pour commencer à utiliser Astro.
+
+Si vous avez manqué l'étape `npm install` durant l'assistant `create-astro`, veuillez vous assurer d'installer les dépendances avant de continuer.
+
+## 2. Lancer Astro ✨
 
 Vous pouvez vous attendre à utiliser le serveur de développement intégré d'Astro pour la plupart de vos projets. C'est de cette manière que vous lancerez votre projet localement pendant le développement.
 
 Pour commencer, utilisez votre gestionnaire de Packages pour lancer le script de démarrage par défaut :
 
-```bash
-# NPM
-npm run dev
-
-# Yarn
-yarn start
-
-# PNPM
-pnpm run dev
-```
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  npm run dev
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  pnpm run dev
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn run dev
+  ```
+  </Fragment>
+</PackageManagerTabs>
 
 Si tout se passe comme prévu, Astro devrait maintenant être en train de faire tourner votre projet à l'addresse [http://localhost:3000/](http://localhost:3000/) !
 
 Astro va également suivre les modifications de fichiers dans le répertoire `src/`, vous n'aurez donc pas besoin de redémarrer le serveur à chaque fois que vous apporterez des modifications au cours du développement.
 
-Si vous n'arrivez pas à ouvrir votre projet dans le navigateur, revenez au terminal où vous avez lancé le script `start` pour voir ce qui a mal tourné.
-
-## 4. Déployer sur le Web
-
-Il est temps de déployer votre projet sur la toile ! Lancez la commande `build` dans votre projet pour construire votre site web statique vers un nouveau répertoire nommé `dist/`.
-
-```bash
-# NPM
-npm run build
-
-# Yarn
-yarn build
-
-# PNPM
-pnpm run build
-```
-
-Quand la commande se termine, vous devriez avoir un nouveau répertoire `dist/` dans votre projet que vous pouvez déployer directement sur votre hébergeur web favori.
-
-Pour commencer à déployer votre site web gratuitement, allez jeter un oeil à notre fier partenaire d'hébergement, [Netlify](https://www.netlify.com/). Pour obtenir des instructions sur la mise en place d'un déploiement, lisez notre [guide de déploiement](/fr/guides/deploy/).
+Si vous n'arrivez pas à ouvrir votre projet dans le navigateur, revenez au terminal où vous avez lancé le script `dev` pour voir ce qui a mal tourné,  ou si votre projet est affiché sur une url différente que celle mentionnée ci-dessus.
```

Comment:
I might be nit-picking here but don't you think "servi" would be more accurate than "affiché" here?

Reply:
ok, served => servi.